testing/test_aux.sh

```diff
@@ -96,9 +96,6 @@ cmd/build inline
 echo Starting aux test run...
 cmd/run -s -k
 
-# Check custom timeout
-cat inst/cmdrun.log | grep "Monitoring timeout for macoui after 1s" | tee -a $TEST_RESULTS
```

Comment:
I think removing this is gonna cause aux test to fail.

Reply:
Whelp. You were right... I could have sworn I checked that.

BTW, the functionality is still tested without this -- it shows up in several places when the macoui (and other tests) fail because of the timeout...